src/lib/esnext.iterator.d.ts

```diff
@@ -0,0 +1,133 @@
+interface NativeIterator<T, TReturn = void, TNext = undefined> extends Iterator<T, TReturn, TNext> {

```

Comment:
The word choose "Native" is not so good. Do we have other options?

Reply:
@rbuckton had previously suggested [`Builtin`](https://github.com/microsoft/TypeScript/issues/54481#issuecomment-1572920703), which I'm also fine with.